taskcluster/app_services_taskgraph/transforms/toolchain.py

```diff
@@ -4,16 +4,37 @@
 
 from __future__ import absolute_import, print_function, unicode_literals
 
+import subprocess
+
 from taskgraph.transforms.base import TransformSequence
 from taskgraph.util.schema import resolve_keyed_by
 
 transforms = TransformSequence()
 
+# TODO: Bug 1637695 to be removed once we retire these old indexes
+TOOLCHAIN_OLD_INDEX = {
```

Comment:
Note: we already have `queue:route:index.project.application-services.*` granted by https://hg.mozilla.org/ci/ci-configuration/file/tip/grants.yml so we should not be hitting scope issues.